optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -205,11 +214,21 @@ public String getSolverRankingClassFullName() {
         return solverRankingClass == null ? null : solverRankingClass.getName();
     }
 
+    /**
+     * Needed for Freemarker Template Language.
+     * @param solverBenchmarkResult the solver benchmark result to get from the map
+     * @return see {@link Map#get(Object)}
+     */
+    public List<File> getSubSingleBenchmarkAggregationChartFileMapEntry(ProblemBenchmarkResult problemBenchmarkResult) {
+        return subSingleBenchmarkAggregationChartFileMap.get(problemBenchmarkResult);
+    }
+
     // ************************************************************************
     // Write methods
     // ************************************************************************
 
     public void writeReport() {
+        logger.info("Generating benchmarking report...");
```

Comment:
Good idea! This feedback to the user has been missing.

Replace "benchmarking report" by "benchmark report", as most of the code/manual calls is the later term.